TestDriver/src/mainScreen.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import { Button, StyleSheet, Text, View } from "react-native";
+import { connect } from "react-redux";
+
+import Heap from "react-native-heap-analytics";
```

Comment:
Nit: should this be in the same block as the the `react` imports, since it's still a module, rather than one of the app's source files?